doc/index.md

```diff
@@ -356,14 +404,28 @@ print(reason_phrases["342"]) --> "Unassigned"
 
 ## http.h1_stream
 
-In addition to following the [*stream*](#stream) interface and the methods from [http.stream_common](#http.stream_common),
-an `http.h1_stream` has the following methods:
+The h1_stream represents an HTTP 1.0 or 1.1 request/response. The module follows the [*stream*](#stream) interface and the methods from [http.stream_common](#http.stream_common), as well as the following HTTP 1 specific functions:
 
 ### `h1_stream:set_state(new)` <!-- --> {#http.h1_stream:set_state}
 
+Sets the h1_stream.state if `new` is one of the following valid states:
```

Comment:
If I read the function correctly, it checks for valid values so I thought it best to show them. I first tried a formatted list, but decided Lua was cleaner.  Up to you if you want me to remove it.